.github/workflows/codeql.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '15 3 * * 4'
```

Comment:
CodeQL Team does recommend this running on a schedule just to track vulnerabilities on a regular basis but maybe we can remove it since out PR rate is high?

Reply:
I think PR rate is high enough